.changeset/shy-carrots-love.md

```diff
@@ -0,0 +1,5 @@
+---
+"changelog": minor
+---
+
+Use Rowan model for CST, add error recovery, refactor parser
```

Comment:
I suggest wording changesets in terms of user-visible changes, rather than for example "refactoring parser", as it won't change the public API/behavior of the parser. For example, mentioning that CST nodes now have a different structure, or not storing ranges on nodes for perf, or tokens carrying the indvididual strings now, etc...
We can also include more than one changeset (markdown file) in this PR, and they will be aggregated as multiple entries in the `CHANGELOG.md`.

Reply:
dpne